jubatus/core/driver/classifier_test.cpp

```diff
@@ -358,28 +359,56 @@ vector<shared_ptr<classifier_base> > create_linear_classifiers() {
   method.push_back(shared_ptr<classifier_base>(
           new core::classifier::perceptron(storage)));
 
+  storage.reset(new core::storage::local_storage_mixture);
```

Comment:
We should consider using type-parameterized tests here.
As this is an existing code, you don't do that in this PR.